### PR TITLE
Replace Anjelique Loveless' Hex-Blaster and Redeemer's Pistol with proper items

### PR DESCRIPTION
**Pull request by Cuingamehtar**
The items she was carrying were modified dueling pistol and arquebus instead of the proper items from the pack.

### CODE REVIEW

File: packs/outlaws-of-alkenstar-bestiary/anjelique-loveless.json [hunk 1]
```diff
@@ -1137,4 +1141,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/outlaws-of-alkenstar-bestiary/anjelique-loveless.json:1144]:
```suggestion
}

```
Don't edit the json files directly, do it in Foundry then extract

**Comment by Cuingamehtar** [comment]
Reexporting data from Foundry also stripped linkedWeapon flags from the attacks. Please check if this is intentional.
I've deliberately kept the item ids the same, even though the items were added to the actor from the pack, as I don't see a reason to change those.

**Comment by simonward** [comment]
> Reexporting data from Foundry also stripped linkedWeapon flags from the attacks. Please check if this is intentional.
> I've deliberately kept the item ids the same, even though the items were added to the actor from the pack, as I don't see a reason to change those.

The `linkedWeapon`s were likely removed when the original weapons were deleted from inventory. These need to be present for `CriticalSpecialization` to function correctly.

**Comment by Cuingamehtar** [comment]
I've manually edited in the flags and weaponType property as I don't see a way to set it in Foundry.

**Comment by TikaelSol** [comment]
I redid the strikes in Foundry with #16071

**Pull request closed without merging**